src/Runner.Common/HostContext.cs

```diff
@@ -647,5 +647,7 @@ public enum ShutdownReason
     {
         UserCancelled = 0,
```

Comment:
Do we no longer use `UserCancelled`?

Reply:
It is used it in several key locations; so it is still needed.

- https://github.com/actions/runner/blob/main/src/Runner.Listener/Runner.cs#L264
- https://github.com/actions/runner/blob/main/src/Runner.Listener/Runner.cs#L291
- https://github.com/actions/runner/blob/main/src/Runner.Listener/JobDispatcher.cs#L576
- https://github.com/actions/runner/blob/main/src/Runner.Worker/JobRunner.cs#L78
- https://github.com/actions/runner/blob/main/src/Runner.Worker/Worker.cs#L120
- https://github.com/actions/runner/blob/main/src/Runner.Worker/Worker.cs#L217